hyrax/app/presenters/hyrax/dataset_presenter.rb

```diff
@@ -7,7 +7,8 @@ class DatasetPresenter < Hyrax::WorkShowPresenter
       :characterization_methods, :computational_methods, :data_origin,
       :complex_instrument, :origin_system_provenance, :properties_addressed,
       :complex_relation, :specimen_set, :complex_specimen_type,
-      :synthesis_and_processing, :custom_property, to: :solr_document
+      :synthesis_and_processing, :custom_property, :first_published_url,
```

Comment:
can't delegate to `:first_published_url` in `:solr_document` because it was added to the publisher model, not `solr_document`.

Reply:
I've added a commit that removes the delegate to solr_document, but since it's being defined twice (publication and dataset) perhaps it should be moved there as may have been intended.